plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -149,6 +150,24 @@ private function add_group_blocks() {
 				)
 			);
 		}
+
+		// Linked Products tabgit.
+		if ( Features::is_enabled( 'product-linked' ) ) {
+			$this->add_group(
+				array(
+					'id'             => $this::GROUP_IDS['LINKED_PRODUCTS'],
+					'order'          => 60,
+					'attributes'     => array(
+						'title' => __( 'Linked products', 'woocommerce' ),
+					),
+					'hideConditions' => Features::is_enabled( 'product-linked' ) ? array(
+						array(
+							'expression' => 'editedProduct.type === "grouped"',
```

Comment:
The AC says this `When the product is a Grouped Product or an External/Affiliate Product, the Cross-sell section should be hidden. Only the Upsell section should appear.`. Meaning that the whole tab should still be visible which is not the case of this line.

Reply:
Thanks for the comment, Mikael. I wonder if we address this visibility logic in a follow-up.

Reply:
Thanks for the comment, Mikael. I wonder if we address this visibility logic in a follow-up.

Reply:
If you want! But please don't forget to remove this since it is not correct.